### ISSUE TITLE
peg syntax error on operator prefix match

### ISSUE THREAD

**Issue by mccanne** [brimdata/zed#3249]
I ran into this ambiguity...
```
% zapi query 'from Raw | topic=="Invoices"'
error parsing Zed at column 15:
from Raw | topic=="Invoices"
          === ^ ===
```
It appears that the PEG parser is matching "top" as an operator then getting stuck parsing the implied filter expression.  Operator names should require whitespace or EOF to avoid this problem.

**Comment by philrz** [comment]
Looks like a duplicate of #3072 BTW.

**Comment by philrz** [comment]
I'm closing this one as a duplicate of #3072, which was indeed fixed by #3792.
